src/deroff.rs

```diff
@@ -538,12 +538,311 @@ impl Deroffer {
         unimplemented!()
     }
 
+    fn quoted_arg(&self) -> bool {
+        unimplemented!()
+    }
+
+    fn request_or_macro(&mut self) -> bool {
+        // self.skip_char();
+        self.s = self.skip_char(&self.s, None).to_owned();
+
+        let s0 = &self.s[1..2];
+
+        match s0 {
+            "\\" => {
+                // if self.str_at(1) == Some('"') {
+                if Self::str_at(&self.s, 1) == "\"" {
+                    // self.condputs("\n");
+                    return true;
+                }
+            }
+            "[" => {
+                self.refer = true;
+                // self.condputs("\n");
+                return true;
+            }
+            "]" => {
+                self.refer = false;
+                // self.skip_char();
+                self.s = self.skip_char(&self.s, None).to_owned();
+                // return self.text();
```

Comment:
Why is this line being left out? It was in the original python code.

Reply:
See replies about `condputs`